### PR TITLE
Cloud changelog Nov 17th

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by spirosoik — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -4,12 +4,49 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
 - [Release 2022-10-13](#release-2022-10-13)
 - [Release 2022-10-06](#release-2022-10-06)
-- [Release 2022-09-15](#release-2022-09-15)
+
+## Release 2022-11-17
+
+### Improvements
+
+#### User Interface (UI)
+ - The “Mark as Unread” option was added to the ``…`` menu for channels in the left-hand side sidebar. Alt+click on a channel on the left-hand side also now marks the last post in the channel as unread.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:19]:
```suggestion
 - The **Mark as Unread** option was added to the ``…`` menu for channels in the left-hand side sidebar. Pressing Alt while selecting a channel on the left-hand side now also marks the last post in the channel as unread.
```

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -4,12 +4,49 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Channel members are now able to remove themselves from a channel via the right-hand side channel members list.
+ - Updated prepackaged boards version to v7.5.0.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:21]:
```suggestion
 - Updated prepackaged Boards version to v7.5.0.
```

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -4,12 +4,49 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Updated prepackaged Calls version v0.10.0.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:22]:
```suggestion
 - Updated prepackaged Calls version to v0.10.0.
```

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -4,12 +4,49 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Downgraded Bulgarian, Persian, and Simplified Chinese language support to Alpha.
+
+#### Administration
+ - The **System Console > File Sharing and Download** setting is now unhidden and honored in Cloud.
+ - **Total Active Users** was renamed to **Total Activated Users** in **system console > Site Statistics**.
+ - Optimized ``ThreadStore.MarkAllAsUnreadByTeam``.
+
+### API Changes
+ - A new API method ``RegisterCollectionAndTopic(collectionType, topicType string) (error)`` was added  to the Plugin API and the following hooks:
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:31]:
```suggestion
 - A new API method ``RegisterCollectionAndTopic(collectionType, topicType string) (error)`` was added to the Plugin API and the following hooks:
```

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -4,12 +4,49 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+    - ``GetAllUserIdsForCollection(c *Context, collectionType, collectionID string) ([]string, error)``
+    - ``GetTopicRedirect(c *Context, topicType, topicID string) (string, error)``
+    - ``GetCollectionMetadataByIds(c *Context, collectionType string, collectionIds []string) (map[string]model.CollectionMetadata, error)``
+    - ``GetTopicMetadataByIds(c *Context, topicType string, topicIds []string) (map[string]*model.TopicMetadata, error)``
+
+### Bug Fixes
+ - Fixed an issue where exports did not contain favorited Direct Message channels.
+ - Fixed an issue where screen readers did not announce search results on "Invite members to channel" modal.
+ - Fixed an issue where screen readers did not announce the status of the user when hovering over the user status icon.
+ - Fixed an issue where users with narrow screens could not see the Profile Setting section.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:43]:
```suggestion
 - Fixed an issue where users with narrow screens could not see the **Profile Settings** section within the **Settings** modal.
```

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -4,12 +4,49 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Fixed an issue where users were unable to access the create account button on narrow screens.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:44]:
```suggestion
 - Fixed an issue where users were unable to access the **Create an account** option on narrow screens.
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

File: source/install/cloud-changelog.md [hunk 7]
```diff
@@ -4,12 +4,49 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
-- [Release 2022-09-15](#release-2022-09-15)
+
+## Release 2022-11-17
+
+### Improvements
+
+#### User Interface (UI)
+ - The **Mark as Unread** option was added to the ``…`` menu for channels in the left-hand side sidebar. Pressing Alt while selecting a channel on the left-hand side now also marks the last post in the channel as unread.
+ - Channel members are now able to remove themselves from a channel via the right-hand side channel members list.
+ - Updated prepackaged Boards version to v7.5.0.
```
Comment (wuwinson) [inline source/install/cloud-changelog.md:21]: @amyblais Did we not want to include the list of Board improvements as well?

Comment (amyblais) [reply source/install/cloud-changelog.md]: Yes, indeed, added now.

File: source/install/cloud-changelog.md [hunk 8]
```diff
@@ -4,12 +4,57 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Optimized ``ThreadStore.MarkAllAsUnreadByTeam``.
+
+### API Changes
+ - A new API method ``RegisterCollectionAndTopic(collectionType, topicType string) (error)`` was added to the Plugin API and the following hooks:
+    - ``UserHasPermissionToCollection(c *Context, userID, collectionType, collectionId string, permission *model.Permission) (bool, error)``
+    - ``GetAllCollectionIDsForUser(c *Context, userID, collectionType string) ([]string, error)``
+    - ``GetAllUserIdsForCollection(c *Context, collectionType, collectionID string) ([]string, error)``
+    - ``GetTopicRedirect(c *Context, topicType, topicID string) (string, error)``
+    - ``GetCollectionMetadataByIds(c *Context, collectionType string, collectionIds []string) (map[string]model.CollectionMetadata, error)``
+    - ``GetTopicMetadataByIds(c *Context, topicType string, topicIds []string) (map[string]*model.TopicMetadata, error)``
```
[the remark this answers is not in the document]
Comment (amyblais) [reply source/install/cloud-changelog.md:45]: @lieut-data Are they behind a feature flag?

**Review by wuwinson — APPROVED** [review]
LGTM!

File: source/install/cloud-changelog.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (lieut-data) [reply source/install/cloud-changelog.md:45]: @amyblais, it's not possible to put the API endpoints themselves behind a feature flag, but they are indeed non-functional without that feature flag.

**Review by mickmister — APPROVED** [review]

**Review by marianunez — APPROVED** [review]

**Review by spirosoik — APPROVED** [review]

**Review by iomodo — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Pull request merged**